src/main/java/ru/mambetpages/dto/GetProfileDto.java

```diff
@@ -0,0 +1,22 @@
+package ru.mambetpages.dto;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Data;
+
+@Data
+public class GetProfileDto {
+    @Schema(name = "Имя пользователя")
+    String name;
+
+    @Schema(name = "Фамилия пользователя")
+    String lastName;
+
+    @Schema(name = "Описание профиля")
+    String description;
+
+    @Schema(name = "Фотография профиля")
+    String photo;
+
+    @Schema(name = "Емайл пользователя")
+    String email;
+}
```

Comment:
I suggest using trailing space. https://softwareengineering.stackexchange.com/questions/121555/why-is-trailing-whitespace-a-big-deal